src/main/resources/db/changelog/changes/create-sharing-policy-table.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+  <changeSet id="MODCONSKC-18@@Implement sharing of authorization policies" author="azizbekxm">
+
+    <createTable tableName="sharing_policy">
+      <column name="id" type="uuid">
+        <constraints primaryKey="true" primaryKeyName="pk_shared_policy_id"/>
+      </column>
+      <column name="policy_id" type="uuid"/>
```

Comment:
Could the field be null?

Reply:
Thanks to point out, can't be, and added nullable validation